modules/ROOT/pages/salesforce/salesforce-mktg-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,112 @@
+
+* Removed support for FuelSDK. The connector now uses the Mule HTTP client to perform the SOAP requests to the API.
+* Changed the supported connection types
+* Changed the Upsert entities operation
+* Removed some operations and grouped their functionality into fewer operations
+
+== Changes in Connection Types
+
+This release contains the following changes to connection types:
+
+* Added support for using OAuth authentication
+* Removed OAuth Username and Password authentication, because the Salesforce Marketing Cloud deprecated this authentication type
+
+== Changed Operations
+
+When using the Upsert entities operation, the connector changed its request type from Update to Create. The name remains the same, only the type of the request has changed.
+
+== Removed Operations
+
+Salesforce Marketing Cloud 3.0.0 has the same functionality as the 2.x versions, but it implements this functionality with fewer operations.
+
+The following operations were removed from Salesforce Marketing Cloud Connector:
+
+// [cols="50,50"]
+[%header%autowidth.spread]
+|===
+|Removed Operation | Functionality Covered By
+
+| Configure Create
+a| Configure action operation with the Action parameter set to `Create`.
+
+| Configure Update
+a| Configure action operation with the Action parameter set to `Update`
+
+| Configure Delete
+a| Configure action operation with the Action parameter set to `Delete`
+
+| Perform Start
+a| Perform action operation with the Action parameter set to `Start`
+
+| Perform Stop
+a| Perform action operation with the Action parameter set to `Stop`
+
+| Perform Get Max Count
+a| Perform action operation with the Action parameter set to  `Get Max Count`
+
+|===
+
+== Changes in Operations Metadata
+
+There are no changes related to keys, input metadata, or output metadata.
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must:
+
+. Create a backup of your files, data, and configuration in case you must restore to the previous version.
+. Install Salesforce Marketing Cloud Connector v3.0.0.
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade from Salesforce Marketing Cloud Connector v2.2.x to Salesforce Marketing Cloud Connector v3.0.0:
+
+. In Anypoint Studio, create a Mule project.
+. In the Mule Palette view, click *Search in Exchange*.
+. In the *Add Dependencies to Project* window, enter `Salesforce Marketing Cloud 3` in the search field.
+. In the *Available modules* section, select *Salesforce Marketing Cloud Connector* and click *Add*.
+. Click *Finish*.
+. Verify that the `mule-sfdc-marketing-cloud-connector` dependency version is `3.0.0` in the `pom.xml` file.
+
+Studio upgrades the connector automatically.
+
+== Verify the Upgrade
+
+After you install the latest version of the connector, follow these steps to verify the upgrade:
+
+. In Studio, verify that there are no errors in the *Problems* or *Console* views.
+. Check the project `pom.xml` file and verify that there are no problems.
```

Comment:
Wasn't this step taken in the previous task?

Reply:
Following the template. We will revisit later on.